docs/source/deployment/kubernetes/corda-kubernetes-deployment-option.rst

```diff
@@ -0,0 +1,175 @@
+Corda Kubernetes Deployment option
+==================================
+
+Now that we have reviewed the architecture and the prerequisites we can start talking about the actual deployment scripts.
+
+The scripts are located at: `Github: Corda Deployments <https://github.com/corda/corda-deployments>`_
+
+The scripts come with usage instructions of their own, but it is worth explaining the steps in more details.
+
+There are a few distinct steps to deploying a Corda Node that are always required.
+
+* The first step is registration / certificates generation.
+* Then moving on to deployment
+* And finally starting everything up and testing
+
+Registration / certificates generation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As discussed on the `Prerequisites <prerequisites.html>`_ page, we need to generate a PKI structure for the Corda Firewall to use.
+The generated certificates from this process should be copied to the following folder: ``helm/files/certificates/firewall_tunnel``. The required files are: ``bridge.jks``, ``float.jks`` and ``trust.jks``.
```

Comment:
Suggestion : new line for second sentence, looks better in the HTML output. 

The generated certificates from this process should be copied to the following folder: ``helm/files/certificates/firewall_tunnel``. 

The required files are: ``bridge.jks``, ``float.jks`` and ``trust.jks``.